numba_dppy/tests/test_array_utils.py

```diff
@@ -19,23 +19,19 @@
 import numpy as np
 import pytest
 
-from numba_dppy.tests._helper import skip_test
+from numba_dppy.tests._helper import filter_strings
 from numba_dppy.utils import (
     as_usm_obj,
     copy_to_numpy_from_usm_obj,
     has_usm_memory,
 )
 
-from . import _helper
-
-
-def test_has_usm_memory(offload_device):
-    if skip_test(offload_device):
-        pytest.skip()
 
+@pytest.mark.parametrize("filter_str", filter_strings)
+def test_has_usm_memory(filter_str):
     a = np.ones(1023, dtype=np.float32)
 
-    with dpctl.device_context(offload_device):
+    with dpctl.device_context(filter_str):
```

Comment:
3 lines changed instead of 1.

Reply:
It's not entirely clear what you mean

Reply:
If old variable name used then added line 
`@pytest.mark.parametrize("offload_device", filter_strings)`
and 2 lines removed:
```python
    if skip_test(offload_device):
        pytest.skip()
```
That is all. Pretty clean change. So I suggest not rename variable.